Makefile

```diff
@@ -27,7 +27,7 @@ fmt: install
 	@$(ESLINT) --fix .
 .PHONY: fmt
 
-test: install
+test: install lint
```

Comment:
At some point we want to turn it back on for everything, so if you make sure it's linted while you're in here, turning it on will save us some time later.